SerialPrograms/Source/PokemonSwSh/Inference/RNG/PokemonSwSh_OrbeetleAttackAnimationDetector.h

```diff
@@ -0,0 +1,46 @@
+/*  Orbeetle Attack Animation Detector
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ *
+ *      Detects whether Orbeetle's animation is special or physical.
+ *
+ */
+
+#ifndef PokemonAutomation_PokemonSwSh_OrbeetleAttackAnimationDetector_H
+#define PokemonAutomation_PokemonSwSh_OrbeetleAttackAnimationDetector_H
+
+#include "CommonFramework/VideoPipeline/VideoOverlay.h"
+#include "CommonFramework/Tools/ConsoleHandle.h"
+
+namespace PokemonAutomation {
+    class BotBaseContext;
+    class ProgramEnvironment;
+    namespace NintendoSwitch {
```

Comment:
I'll start with formatting. lol

Don't need to indent namespaces because that indents the entire file.

Reply:
Formatting should be fixed now.